lib/puppet/pops/types/type_factory.rb

```diff
@@ -248,7 +248,11 @@ def self.tuple(types = [], size_type = nil)
   # @api public
   #
   def self.boolean(value = nil)
-    value.nil? ? PBooleanType::DEFAULT : (value ? PBooleanType::TRUE : PBooleanType::FALSE)
+    if value.nil?
+      PBooleanType::DEFAULT
+    else
+      (value ? PBooleanType::TRUE : PBooleanType::FALSE)
```

Comment:
I don't think this still needs to be in parentheses now that it's not part of a nested ternary operator